### ISSUE TITLE
please tag a new release/version

### ISSUE THREAD

**Issue by jnahmias** [FirefighterBlu3/python-pam#20]
I think there's been enough commits/changes since v1.8.4 [Jun 16, 2018] to justify a minor version bump to v1.8.5.

**Comment by FirefighterBlu3** [comment]
I've pushed a pending PR and have some more changes and I'll bump again. The additional changes are significant so I'll probably do a major bump

**Comment by FirefighterBlu3** [comment]
2.0 release is in the works

**Comment by jnahmias** [comment]
@FirefighterBlu3 any updates you can share on this?

**Comment by FirefighterBlu3** [comment]
Yes! I am working on a 2.0.0 branch for release. I have some things to tidy up and some testing to complete and it'll ship

**Comment by jnahmias** [comment]
Sounds good. Anything I can do to help?

**Comment by abompard** [comment]
Would it be possible to push a 1.8.5 version before 2.0.0 is ready to be published?

**Comment by FirefighterBlu3** [comment]
@abompard my apologies. what do you need in a 1.8.5 vs. 2.0.0?

**Comment by FirefighterBlu3** [comment]
2.0.1 will be released shortly pending a fix confirmation

**Comment by abompard** [comment]
Basically just API compatibility, it would be easier to upgrade to 1.8.5 than to 2.0.0. But it's up to you of course.

**Comment by FirefighterBlu3** [comment]
does the commit for #31 work for you? it should make things fully backward
compatible

**Comment by FirefighterBlu3** [comment]
2.0.2 out, includes some additional testing

**Comment by pmorch** [comment]
Can this be closed?

**Comment by MaKaNu** [comment]
> Can this be closed?

I would tend to say yes. But I spot a lot issues with building the project in the state right now. 

Examples:
- #3
- The Arch PKGBUILD revers a setup.py, which is not present since 1.8.4

I just jumped onto the project and have no clue what direction the maintainer wants to achieve, so I cannot provide a Solution right now

**Comment by jvoisin** [comment]
A new version would be appreciated, if only so that linux distributions (like [Alpine](https://gitlab.alpinelinux.org/alpine/aports/-/work_items/17822) and [Debian](https://wiki.debian.org/Python3-six-removal)) can (finally) get rid of the dependency on `six`.
